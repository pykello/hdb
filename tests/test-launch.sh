#!/bin/sh

hdb_port_string=`./launch-hdb.py 2 10 100`
read -a ports <<< $hdb_port_string

./stop-hdb.py ${ports[0]}

