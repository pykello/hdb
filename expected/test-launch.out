Shutting down (localhost, ${ports[0]}) ...
OK.
Shutting down (localhost, ${ports[1]}) ...
OK.
